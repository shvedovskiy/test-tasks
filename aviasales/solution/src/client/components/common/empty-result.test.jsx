/* eslint-disable no-undef */
import React from 'react';

import EmptyResult from './empty-result';


describe('<EmptyResult />', () => {
  const emptyResult = shallow(<EmptyResult />);

  it('renders properly', () => {
    expect(emptyResult).toMatchSnapshot();
  });
});
