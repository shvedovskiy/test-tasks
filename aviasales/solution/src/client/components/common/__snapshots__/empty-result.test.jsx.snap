// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyResult /> renders properly 1`] = `
<Message>
  <h2>
    По заданным критериям билетов не найдено!
  </h2>
</Message>
`;
